meson.build

```diff
@@ -78,6 +74,19 @@ lib_install_dir = get_option('libdir')
 lib_gstd_dir = []
 lib_gstc_dir = []
 
+# Define compiler args and include directories
+runstatedir = get_option('with-gstd-runstatedir') + '/'
+if runstatedir == '${prefix}/var/run/gstd/'
+   runstatedir = prefix + '/var/run/gstd/'
```

Comment:
Isn't this equivalent?

Reply:
No, in meson the ${prefix} does not get replaced as in autoconf I think.